app/models/manageiq/providers/network_manager.rb

```diff
@@ -9,6 +9,7 @@ class << model_name
       define_method(:singular_route_key) { "ems_network" }
     end
 
+    supports_not :create
     supports_not :ems_network_new
```

Comment:
Actually we could probably drop `ems_network_new` also here, looks like it is used by the UI to check if edit should be allowed but we could simply use `supports?(:update)` instead

Reply:
lol. do we have an emoji for scope creep?

I'll create a separate PR. it will conflict with this PR but it looks straight forward enough.
I do want to move these over to CRUD, but right now, my goal is getting us to `supports` (in any form)

Reply:
Yeah definitely you can do that in a different PR, just caught my eye here since `supports :create` and `supports :ems_network_new` ostensibly track the same feature.

Reply:
That was my first impression but now seeing how it is used (per https://github.com/ManageIQ/manageiq/pull/21573/files#r756180425) I think `s/ems_network_new/update`

Reply:
Correct, number B :wink: